invenio_rdm_records/services/review/service.py

```diff
@@ -143,18 +144,44 @@ def delete(self, identity, id_, revision_id=None, uow=None):
         return True
 
     @unit_of_work()
-    def submit(self, identity, id_, data=None, revision_id=None, uow=None):
-        """Submit record for review."""
-        # Get record and check permission
+    def submit(
+        self, identity, id_, data=None, direct_publish=None, revision_id=None, uow=None
+    ):
+        """Submit record for review or direct publish to the community."""
         draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-        self.require_permission(identity, "update_draft", record=draft)
-
         # Preconditions
         if draft.parent.review is None:
             raise ReviewNotFoundError()
 
         # since it is submit review action, assume the receiver is community
         resolved_community = draft.parent.review.receiver.resolve()
+        if direct_publish:
+            # Direct publish to community workflow
+            current_communities.service.require_permission(
+                identity, "direct_publish", record=resolved_community
+            )
+
+            # Submit draft for review
+            submitted_review = self._submit(
```

Comment:
@slint after our chat of yesterday, for example, here, we might not want to trigger: an e-mail for the creation of the request, to owner and community curators, and then immediately after a second e-mail that it has been accepted.
It might make sense to move the `send_notif` at the service level, and not the at the action level.